trunk/SUAVE/Optimization/Nexus.py

```diff
@@ -201,12 +201,23 @@ def inequality_constraint(self,x = None):
         if iqconstraints == []:
             scaled_constraints = []
         else:
-            constraint_values = help_fun.get_values(self,iqconstraints,aliases)
-            constraint_values[iqconstraints[:,1]=='<'] = -constraint_values[iqconstraints[:,1]=='<']
-            bnd_constraints   = constraint_values - help_fun.scale_const_bnds(iqconstraints)
-            scaled_constraints = help_fun.scale_const_values(iqconstraints,constraint_values)
 
-        return scaled_constraints      
+            # get constaint values 
+            constraint_values = help_fun.get_values(self,iqconstraints,aliases)          
+            
+            # scale bounds 
+            scaled_bnd_constraints  = help_fun.scale_const_bnds(iqconstraints)
+            
+            # scale constaits 
+            scaled_constraints = help_fun.scale_const_values(iqconstraints,constraint_values)
+            
+            # determine difference between bounds and constaints 
+            constraint_evaluations = scaled_constraints  - scaled_bnd_constraints
+            
+            # coorect constaints based on sign 
+            constraint_evaluations[iqconstraints[:,1]=='<'] = -constraint_evaluations[iqconstraints[:,1]=='<']
+            
+        return constraint_evaluations       
```

Comment:
I'm confused by this change. The old order of operations appears to be:

1. Read in inequality constraint values
2. Reorient less-than constraints to be greater-than constraints
3. Determine scaled value bounds
4. Create bound-offset constraints
5. Scale constraint values

It actually appears to me like it uses the reoriented, but not bound-offset constraint values in Step 5 since the call on line 207 was using `constraint_values,` not `bnd_constraints`

The new order of operations appears to be:

1. Read in inequality constraint values
2. Create scaled value bounds
3. Determine constraint bounds
4. Scale constraint values
5. Reorient less-than constraints to be greater-than constraints

These seem like they should be equivalent (ignoring the bound-offset weirdness) so can you explain why the switch up? I think we discussed this in person but I don't remember the details.

Reply:
If you look in the old code...it never used the bounds i.e. like 206 is pointless